### DISCUSSION TITLE
Community Call Fri Nov 3 at 12noon Eastern

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#445] [Announcements]
Agenda:

- STDC 2024 proposed Workshop or Special Session on "Cyberinfrastructure/Computing for Topology and Dynamics"
    - See https://github.com/pi-base/data/issues/459
- LatticeMeet: https://latticemeet.vercel.app/login

Zoom: https://southalabama.zoom.us/j/93833246214

**Comment by ccaruvana** [comment]
I will be on an airplane around noon on the 20th. I unfortunately also have travel plans the following weekend. Would it be at all possible to move this call to either Monday 16 Oct or Tuesday 17 Oct? We have Fall Break here those two days so my schedule would be wide open for whatever is most convenient for folks, if this is possible.

**Comment by lyengulalp** [comment]
I'm free before 10am, 12-1, or 2-3 on the 16th. Then 9:30 -11 or 12-3 on
the 17th. The 20th I'm pretty open after 9:30am.

**Comment by StevenClontz** [comment]
@prabau would any time Lynne suggested for Mon Oct 16 work for you? They seem to work for me and Chris.

**Reply by prabau** [reply]
How about 2-3 on Oct 16?

**Comment by StevenClontz** [comment]
Great - see everyone on Monday Oct 16 at 2pm Eastern.

**Comment by StevenClontz** [comment]
I don't see anything on our agenda, and a conflict arose on my end and I'm unavailable. So unless anyone else has business to discuss without me, I'll propose delaying our next meeting for a couple weeks.

By the way, if 2pm Eastern Mondays is okay in general, we could keep this timeslot. But maybe it only worked for @ccaruvana due to his fall break?

**Reply by ccaruvana** [reply]
Yeah, I typically have class meetings on Mondays at 2pm, but Fall Break opened these two days up for me. Fridays are still generally open, I just have two family-related travel weekends the two consecutive weekends of 20 and 27 Oct.

**Reply by StevenClontz** [reply]
Then moving back to Fridays at 12noon Eastern this semester works for me, perhaps on Fri Nov 3?

**Reply by prabau** [reply]
Would you have anything available in the afternoon?

**Reply by StevenClontz** [reply]
Updated to correct my previous comment to say 12noon Eastern, 11am Central.

**Reply by StevenClontz** [reply]
I could also do 1pm Eastern that day, but I think 12noon Eastern was the time we had been using.

**Reply by prabau** [reply]
12noon or 1pm Eastern is fine with me

**Reply by StevenClontz** [reply]
@lyengulalp can pipe up if it's not a good time, but let's pencil in 12noon Eastern Fri Nov 3.

**Reply by ccaruvana** [reply]
Sounds good to me 👍🏼

**Comment by StevenClontz** [comment]
Reminder of our community call scheduled for Friday. I have one potential agenda item - edit the post at the root or reply here with any other suggestions.

**Reply by lyengulalp** [reply]
I'll be there from 12 to 12:29 (have another meeting at 12:30).

**Comment by prabau** [comment]
@StevenClontz One thing occurred to me.  For your worshop at STDC 2024, are you planning to have just one PR (#459) that people would work on, or were you thinking several small PR that people would work on in a few teams?  If the latter, we could do a list of low hanging fruits that folks could choose from, little improvements that could be done in a short amount of time and illustrate well how to contribute to pi-base. (And we would refrain from doing those ourselves for now).  It all depends on what you were planning.

**Reply by StevenClontz** [reply]
My instinct is that a single new space is probably the right scope for a 75ish minute workshop. But if we want to create some issues and tag them as "good first contribution" I can point to those as a call to action after the workshop, or dip into them if we need more content.
